src/connected/interfaces.ts

```diff
@@ -17,7 +18,7 @@ export type DataLoader = (seriesIds: SeriesId[],
                           context?: any) => TBySeriesId<Promise<LoadedSeriesData>>;
 export interface DebugStoreHooks {
   middlewares?: Function[];
```

Comment:
Ooh, while you're here, can you change this to be more accurate a type than `Function`?